config/secrets.yml

```diff
@@ -1,8 +1,32 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Should this file be in the public repo?

Reply:
Yep, it only includes the 'secret' keys for development and test, which aren't sensitive.